### PR TITLE
Add tooltip to turn button

### PR DESCRIPTION
**Pull request by TheSilentOne1**
providing information about the sequence of turn processing

### CODE REVIEW

**Comment by Zireael07** [comment]
I don't think that an average player needs that sort of information, though.

**Comment by TheSilentOne1** [comment]
In our current LT game, o01eg asked about it, and it does matter in some situations.

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -3194,6 +3194,20 @@ Unable to create persistent_config.xml, check log file for details.
 MAP_BTN_TURN_UPDATE
 Turn %1%
 
+MAP_BTN_TURN_TOOLTIP
+Start Turn
+
+MAP_BTN_TURN_TOOLTIP_DESC
+'''Starts turn processing with the following sequence:
```
Comment (geoffthemedio) [inline default/stringtables/en.txt:3205]: Consider observers, moderators, and when waiting for other players or the server...?

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -3194,6 +3194,20 @@ Unable to create persistent_config.xml, check log file for details.
...
+
+<u>Order execution</u>
+Fleet movement, Colonisation, Invasion, Bombardment etc.
+
+<u>Combat resolution</u>
+
+<u>Production and Research</u>
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: break down into more details / steps?

Comment (TheSilentOne1) [reply default/stringtables/en.txt]: Done, open for suggestions concerning the combat phase.

File: default/stringtables/en.txt (hunk 1 above)
Comment (TheSilentOne1) [reply default/stringtables/en.txt:3205]: Done except for moderators, does the turn button behave differently for them?

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -3198,6 +3198,52 @@ Turn %1%
...
+Turn processing will be executed with the following sequence:
+
+<u>Order execution</u>
+Colonisation, invasion, bombardment, gifting, scrapping and fleet movement
+
+<u>Combat resolution</u>
+
+<u>Production, Research and Growth</u>
+Effect execution, production of ships and buildings, unlocking of new technologies, population growth, update of supply propagation
+'''
```
[the remark this answers is not in the document]
Comment (TheSilentOne1) [reply default/stringtables/en.txt]: Thought so as well, but where should I place it?

[the remark this answers is not in the document]
Comment (TheSilentOne1) [reply default/stringtables/en.txt]: Put it after the other turn button entries for now.

**Pull request merged**